pkg/sdn/plugin/ovscontroller.go

```diff
@@ -306,8 +306,16 @@ func (oc *ovsController) SetPodBandwidth(hostVeth string, ingressBPS, egressBPS
 	return nil
 }
 
-func getPodDetailsByContainerID(flows []string, containerID string) (int, string, string, string, error) {
-	note, err := getPodNote(containerID)
+func getPodDetailsBySandboxID(ovsif ovs.Interface, sandboxID string) (int, string, string, string, error) {
+	flows, err := ovsif.DumpFlows()
```

Comment:
Maybe we should rebase the real pkg/util/ovs implementation on top of the fake one, so that, eg, when you do an AddFlow(), it calls ovs-ofctl *and* also updates its own internal array of flows, so that we can implement code like this without actually having to call dump-flows all the time?